src/utils/live-tunnel.js

```diff
@@ -58,70 +55,38 @@ async function connectTunnel(session, netlifyApiToken, localPort, log) {
 }
 
 async function installTunnelClient(log) {
-  const win = isWindows()
-  const binPath = path.join(os.homedir(), '.netlify', 'tunnel', 'bin')
-  const execName = win ? 'live-tunnel-client.exe' : 'live-tunnel-client'
-  const execPath = path.join(binPath, execName)
-  const newVersion = await fetchTunnelClient(execPath)
-  if (!newVersion) {
+  const binPath = getPathInHome(['tunnel', 'bin'])
+  const shouldFetch = await shouldFetchLatestVersion({
+    binPath,
+    packageName: PACKAGE_NAME,
+    execArgs: ['version'],
+    pattern: `${PACKAGE_NAME}\\/v?([^\\s]+)`,
+    execName: EXEC_NAME,
+  })
+  if (!shouldFetch) {
     return
   }
 
   log(`${NETLIFYDEVLOG} Installing Live Tunnel Client`)
 
-  const platform = win ? 'windows' : process.platform
-  const extension = win ? 'zip' : 'tar.gz'
-  const release = {
-    repository: 'netlify/live-tunnel-client',
-    package: `live-tunnel-client-${platform}-amd64.${extension}`,
+  await fetchLatestVersion({
+    packageName: PACKAGE_NAME,
+    execName: EXEC_NAME,
     destination: binPath,
-    extract: true,
-  }
-  await fetchLatest(release)
-}
-
-async function fetchTunnelClient(execPath) {
-  if (!execExist(execPath)) {
-    return true
-  }
-
-  const { stdout } = await execa(execPath, ['version'])
-  if (!stdout) {
-    return false
-  }
-
-  const match = stdout.match(/^live-tunnel-client\/v?([^\s]+)/)
-  if (!match) {
-    return false
-  }
-
-  return updateAvailable('netlify/live-tunnel-client', match[1])
-}
-
-function execExist(binPath) {
-  if (!fs.existsSync(binPath)) {
-    return false
-  }
-  const stat = fs.statSync(binPath)
-  return stat && stat.isFile() && isExe(stat.mode, stat.gid, stat.uid)
+    extension: process.platform === 'win32' ? 'zip' : 'tar.gz',
+  })
 }
 
-function isExe(mode, gid, uid) {
-  if (isWindows()) {
-    return true
-  }
+const startLiveTunnel = async ({ siteId, netlifyApiToken, localPort, log }) => {
```

Comment:
`createTunnel` and `connectTunnel` were called separately in the dev command.
I grouped them into  `startLiveTunnel`